docs/reference/api-v1alpha.md

```diff
@@ -916,4 +916,540 @@ curl -i \
+
+```
+GET {org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id&target_name=:target_name
+```
+
+**Params**
+
+- `project_id` (**required**) - The UUID of the project to which the deployment target is linked.
+- `target_name` (**required**) - The name of the deployment target.
+
+**Response**
+
+```json
+HTTP status: 200
+
+[
+  {
+    "url": "project_1234.zyx",
+    "updated_by": "02984c87-efe8-4ea1-bcac-9511a34a3df3",
+    "updated_at": "2023-06-07T09:24:04.000000Z",
+    "subject_rules": [
+      {
+        "type": "ANY",
+        "subject_id": ""
+      }
+    ],
+    "state_message": "",
+    "state": "USABLE",
+    "project_id": "a426b4db-1919-483d-926d-06ba1320b209",
+    "organization_id": "7304b7f9-7482-46d4-9b95-3cd5a6ef2e6f",
+    "object_rules": [
+      {
+        "type": "BRANCH",
+        "pattern": "",
+        "match_mode": "ALL"
+      },
+      {
+        "type": "TAG",
+        "pattern": "",
+        "match_mode": "ALL"
+      },
+      {
+        "type": "PR",
+        "pattern": "",
+        "match_mode": "ALL"
+      }
+    ],
+    "name": "DTName123",
+    "last_deployment": null,
+    "id": "e79d3d1c-61cc-4e07-ba32-f86694f5e80d",
+    "description": "DT description",
+    "created_by": "02984c87-efe8-4ea1-bcac-9511a34a3df3",
+    "created_at": "2023-06-01T10:07:36.000000Z",
+    "bookmark_parameter3": "",
+    "bookmark_parameter2": "",
+    "bookmark_parameter1": "",
+    "active": true
+  }
+]
+```
+
+**Example**
+
+```
+curl -i -H "Authorization: Token {api_token}" \
+     "https://{org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id&target_name=:target_name"
+```
+
+### Creating a target
+
+This API endpoint allows you to create a new deployment target and assign it to a specific project.
+
+```
+POST {org_name}.semaphoreci.com/api/v1alpha/deployment_targets?project_id=:project_id
+```
+
+**Request Body**
+
+The request body should be a JSON object, encapsulating details about the deployment target to be created. The available fields are as follows:
+
+  - `name` (**required**) - Unique name for the target within the project.
+  - `project_id` (**required**) - UUID of the project.
+  - `unique_token` (**required**) - Idempotency UUID token.
+  - `description` (*optional*) - A description of the target.
+  - `url` (*optional*) - The URL of the target.
+  - `bookmark_parameter1`, `bookmark_parameter2`, `bookmark_parameter3` (*optional*) - Bookmark parameters.
+  - `subject_rules` (*optional*) - A list of subject rules structured as `{"type": RULE_TYPE, "subject_id": ID}`. The `RULE_TYPE` can be one of `"ANY"`, `"USER"`, `"ROLE"`, `"GROUP"`, `"AUTO"` and the `ID` should be the name of the role or group (which must be valid for the project) or UUID of a user. For `USER` type you can use `{"type": "USER", "git_login": GITLOGIN}` where `GITLOGIN` is a user's git handle. The user must be assigned to the project to be used in the rule.
```

Comment:
Let's remove `"GROUP"` here until it is fully implemented.
On the other hand, we should add a line that describes what  `"AUTO"` is (if auto-promotions are allowed) and how to use it.